central/localscanner/certificates.go

```diff
@@ -56,7 +76,15 @@ func localScannerCertificatesFor(serviceType storage.ServiceType, namespace stri
 }
 
 func generateServiceCertMap(serviceType storage.ServiceType, namespace string, clusterID string) (secretDataMap, error) {
-	if serviceType != storage.ServiceType_SCANNER_SERVICE && serviceType != storage.ServiceType_SCANNER_DB_SERVICE {
+	supportedServices := set.NewFrozenSet(
```

Comment:
maybe we should make this a global `var`?

Reply:
done